pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/AnswerDao.java

```diff
@@ -49,212 +47,38 @@
-            case BOOLEAN:
-                return getJdbiBooleanAnswer().findByAnswerId(answerId);
-            case PICKLIST:
-                return getPicklistAnswerDao()
-                        .findByAnswerId(answerId)
-                        .orElseThrow(() -> new DaoException("Could not find picklist answer with id " + answerId));
-            case TEXT:
-                return getJdbiTextAnswer().findByAnswerId(answerId);
-            case DATE:
-                return getJdbiDateAnswer().getById(answerId).orElseThrow(
-                        () -> new DaoException("Could not find date answer with id " + answerId));
-            case NUMERIC:
-                return getJdbiNumericAnswer().findById(answerId).orElseThrow(
-                        () -> new DaoException("Could not find numeric answer with id " + answerId));
-            case AGREEMENT:
-                AgreementAnswerDto dto = getJdbiAgreementAnswer().findDtoById(answerId).orElseThrow(
-                        () -> new DaoException("Could not find agreement answer with id " + answerId));
-                return new AgreementAnswer(answerId, dto.getQuestionStableId(), dto.getAnswerGuid(), dto.getAnswer());
-            case COMPOSITE:
-                Optional<CompositeAnswerSummaryDto> optionalAnswerSummary =
-                        getJdbiCompositeAnswer().findCompositeAnswerSummary(answerId);
-                if (optionalAnswerSummary.isPresent()) {
-                    CompositeAnswerSummaryDto summaryObj = optionalAnswerSummary.get();
-                    CompositeAnswer answer = new CompositeAnswer(summaryObj.getId(), summaryObj.getQuestionStableId(),
-                            summaryObj.getGuid());
-                    summaryObj.getChildAnswers().forEach((List<AnswerDto> rowChildDtos) -> {
-                        List<Answer> rowOfAnswers = rowChildDtos.stream()
-                                .map(childAnswerDto ->
-                                        //updated query gives us the question information if row exists but answer does not
-                                        //check for null then
-                                        childAnswerDto.getId() == null ? null : getAnswerById(childAnswerDto.getId()))
-                                .collect(Collectors.toList());
-                        answer.addRowOfChildAnswers(rowOfAnswers);
-                    });
-                    return answer;
-                } else {
-                    throw new DaoException("Unable to find CompositeAnswer with id" + answerId);
-                }
-            default:
-                throw new DaoException("Unhandled question type " + questionType);
-        }
+    // Convenience method for creating answers using guids. Prefer the other method that takes ids.
+    default Answer createAnswer(String operatorGuid, String instanceGuid, Answer answer) {
+        long operatorId = getHandle().attach(JdbiUser.class).getUserIdByGuid(operatorGuid);
```

Comment:
Don't know what to do about this id vs guid business. Ideally we won't have to do these extra queries just to convert between id/guid, but not sure what else to do here.